sbt

```diff
@@ -189,7 +189,10 @@ acquire_sbtn () {
   local target="$p/sbtn"
   local archive_target=
   local url=
-  local arch="x86_64"
+  local arch
+  if [[ "$OSTYPE" == "darwin"* ]]; then
+    arch="universal"
+  fi
```

Comment:
There's 

```bash
  elif [[ "$OSTYPE" == "darwin"* ]]; then
```

at line 205, so maybe move that in there? also this means `arch` would be unset on Windows?

Reply:
Thanks for the feedback! that's better solution :)